src/components/LogFilter.jsx

```diff
@@ -90,11 +90,100 @@ const LogFilter = ({ setState, state, fetchFilteredLogs }) => {
     });
   };
 
+  const handleOriginChange = (e) => {
+    const { value } = e.target;
+
+    if (value === 'All Logs') {
+      setState({
+        ...state,
+        logOrigin: 'All Logs',
+      });
+    } else if (value === 'Backend Logs') {
+      setState({
+        ...state,
+        logOrigin: 'Backend Logs',
+      });
+    } else if (value === 'Frontend Logs') {
+      setState({
+        ...state,
+        logOrigin: 'Frontend Logs',
+      });
+    }
+  };
+
   return (
     <div className=' bg-less-dark mr-3 ml-3 mt-5 pr-2 pl-2 rounded flex items-center justify-between shadow-md shadow-extra-dark'>
       <h3 className='mr-5 text-md font-medium font-semibold text-teal'>
         Filter logs
       </h3>
+
+      <div>
+        <Menu
+          as='div'
+          className='relative inline-block text-left py-2.5 px-2.5 ml-2'>
+          <div>
+            <Menu.Button className='inline-flex justify-center w-full px-4 py-2 text-md font-medium text-teal bg-black rounded-md bg-opacity-20 hover:bg-opacity-30 focus:outline-none focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75'>
+              {state.logOrigin ? state.logOrigin : 'Log Origin'}
+              <BiChevronDown
+                className='w-5 h-5 ml-2 -mr-1 text-violet-200 hover:text-violet-100'
+                aria-hidden='true'
+              />
+            </Menu.Button>
+          </div>
+          <Transition
+            as={Fragment}
+            enter='transition ease-out duration-100'
+            enterFrom='transform opacity-0 scale-95'
+            enterTo='transform opacity-100 scale-100'
+            leave='transition ease-in duration-75'
+            leaveFrom='transform opacity-100 scale-100'
+            leaveTo='transform opacity-0 scale-95'>
+            <Menu.Items className='absolute right-0 w-50 mt-2 origin-top-right bg-less-dark divide-y divide-gray-100 rounded-md shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none'>
+              <div className='px-1 py-1 '>
+                <Menu.Item>
+                  {({ active }) => (
+                    <button
+                      className={`${
+                        active ? 'bg-violet-500 text-teal' : 'text-teal'
+                      } group flex rounded-md items-center w-full px-2 py-2 text-sm`}
+                      value='All Logs'
```

Comment:
I forrgot to mention. In here remember instead of using the string to use constantsObject.all whatever de name is and change the value in that object property for the 'all' string instead of true so we only need to change things in one place for everything.